Tasks/DockerV1/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 1,
-        "Patch": 5
+        "Patch": 13
     },
     "demands": [],
     "releaseNotes": "Simplified the task by:<br/>&nbsp;- Providing an option to simply select or type a command.<br/>&nbsp;- Retaining the useful input fields and providing an option to pass the rest as an argument to the command.",
```

Comment:
Version should be 6. I had made a mistake in task versions in master which is why it is 13 there. I don't think it needs to be 13 here

Reply:
Okay - That explains the jump. I'll change it back to 6 then, assuming that you would fix things up in master?